src/lib/phdi-building-blocks/tests/test_standardize.py

```diff
@@ -4,18 +4,19 @@
 
 
```

Comment:
Similar to my comment on `test_geo.py` I think we should add explicit tests for all functions, even ones we wrap inside others that are tested.

Reply:
I agree with this, but at this point I think we should consider it outside the scope of this PR, which focuses on human-centered design changes. I'll create a series of tickets that represent expanding test coverage in each of our modules.